Mod2/Lessons/Week1/VersionControlAndSoloGit.md

```diff
@@ -0,0 +1,113 @@
+>1. [Creating Branches](https://www.loom.com/share/d675a00c7c634d35b977a7df6972b13c)
+>2. [Merging Branches](https://www.loom.com/share/d12d3d510da9402092fe793b4a398bd1)
+>3. [Merged Commit History](https://www.loom.com/share/e5baba208ec441a6a157a8b956dc30f9)
+
+After you have completed the individual work above:
+
+> In small groups: discuss the process that you just completed.  As a starter, answer the following questions:
+> * What is the purpose of branching?
+> * When we merge branches together, why might they not appear in chronological order in the commit history?
+> * What causes a merge conflict?
+
+** Instructor Note ** After letting groups discuss their challenges and questions, do a walkthrough of the process, answering questions as you go.
+
+### GitHub
+So far, we have created a project with a git repository that exists _only_ on our local machines.  This is fine for some personal projects, but most of the time, you are going to want to share your code.  When you share your code, you can get feedback and help a lot more easily.  Also, sharing your code allows for easier collaboration!
+
+**Github** is where we will share our git repositories.  You have used Github to fork and clone existing repositories - now let's create a Github repository of your own.  When this repository is created, you will have two versions of your project:
+* **local** repository that lives only on your computer
+* **remote** repository that lives in the cloud - it can be cloned by anyone on Github.
+
+> Individually, watch the video below to create a GitHub Repository for your Git Practice project.
+> [Creating and Pushing to a Github Repository](https://www.loom.com/share/dba188a5f13e43059d9a84b630280c53)
+
+After you have completed the individual work above:
+
+> In small groups: discuss the process that you just completed.  As a starter, answer the following questions:
+> * When you pushed to Github, did all of your branches get pushed?
+> * Will you only push a project to Github once?
+
+** Instructor Note ** After their discussion, we want to re-inforce that the work they do locally will not appear on github until they push that work up.  There is no 'open channel' between the two repositories.
+
+### A Quick Note!
+In this lesson, we have covered a lot - and we have only scratched the surface of Git and version control processes.  As a developer you will continue to practice and improve these skills, so don't be discouraged if your first few tries with git don't go as planned!  There are very few Git experts in the world, and it is not something you must _master_ in order to be a great developer.  But, you do need to practice git, and ask questions when you get stuck 🙂.
+
+## Check for Understanding
+* In your own words, what is git?  What is Github?
```

Comment:
Maybe add to these questions, What is git and why do we use it? What is Github and why do we use it?